spec/unit/handler_spec.rb

```diff
@@ -45,6 +45,39 @@
       "Logging                  \n"].join)
   end
 
+  it "defaults message_padding to 25 spaces" do
+    logger = TTY::Logger.new(output: output)
+    message = "Logging"
+    expected_padding = 25 - message.size
```

Comment:
would you rather inline these spaces literally into the spec instead of interpolating? I could go either way.

Reply:
I tend to lean on the inline side of this argument. But I like what you've done so don't worry about it - I will accept either way.

As for looking at the output comparison, I often split strings to 'highlight' and make it easier to read output:

```ruby
expect(output.string).to eq(
  "\e[32m#{styles[:info][:symbol]}\e[0m \e[32minfo\e[0m    " \
  "Logging#{' ' * expected_padding}\n"
)
```